### PR TITLE
Upgrading gatekeeper image to v3.6.0

### PR DESCRIPTION
**Pull request by nandajavarma**

### CODE REVIEW

**Comment by ralgozino** [comment]
Hi @nandajavarma !

I see that the PR changes only the image tag. Have you checked if there are changes to the manifests that need to be ported from upstream?

Reading [upstream's release notes](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.6.0) I see some updates that may affect the manifests, like these two:
- ConstraintTemplate CRD moves to v1 🎉
- Helm enable to config controller manager & audit port 🎊

Thanks!

**Comment by nandajavarma** [comment]
Hey Ramiro! I'm testing the changes locally. I'll update it soon.

File: katalog/gatekeeper/core/deploy.yml [hunk 1]
```diff
@@ -33,6 +33,8 @@ spec:
         - --operation=audit
         - --operation=status
         - --logtostderr
+        - --health-addr=:9090
+        - --prometheus-port=8888
```
Comment (angelbarrera92) [inline katalog/gatekeeper/core/deploy.yml:37]: Does it makes sense to create a `ServiceMonitor` to monitor the gatekeeper deployment¿? Or maybe was already there and need to be changed the port from where to scrape metrics¿?

Comment (angelbarrera92) [reply katalog/gatekeeper/core/deploy.yml:37]: I say nothing. Everything looks fine here

**Review by angelbarrera92 — APPROVED** [review]
LGTM!

**Pull request merged**